scripts/test_docker.sh

```diff
@@ -2,10 +2,10 @@
 set -xeuo pipefail
 
 function nf_test() {
-  nextflow run . -profile testing "$@"
+  nextflow run . -profile travis "$@"
```

Comment:
Since we have this wrapper, why don't we use it also for buildReference? with something like
```
nextflow run "$@" -profile travis
nf_test buildReferences.nf  --download 
nf_test . <every other tests>
```